tests/profile_service/index.spec.js

```diff
@@ -33,20 +33,19 @@ describe('profileService', () => {
       it('should get the receipt', (done) => {
         getReceipt('a-test-token', privateKeyFile, 'stub-app-id')
           .then((receipt) => {
-            const profile = receipt.getUserProfile();
-            const outcome = receipt.getOutcome();
-
-            expect(profile).not.toBe(undefined);
             expect(receipt.getReceiptId()).toBe(receiptId);
             expect(receipt.getRememberMeId()).toBe(rememberMeId);
             expect(receipt.getParentRememberMeId()).toBe(parentRememberMeId);
             expect(receipt.getTimestamp()).toBeInstanceOf(Date);
             expect(receipt.getTimestamp().toUTCString()).toBe('Tue, 19 Jul 2016 08:55:38 GMT');
-            expect(profile.phoneNumber).toBe(phoneNumber);
-            expect(`data:image/jpeg;base64,${profile.selfie.toString('base64')}`).toBe(selfie);
-            expect(receipt.getBase64SelfieUri()).toBe(selfie);
```

Comment:
Are we not testing this one anymore?

Reply:
The profile is exposing sugar methods to access each attribute, or you can get the list of attributes... but there is no longer any properties on the profile like `profile.phoneNumber` (mainly due to the support of multiple attribute sharing the same name).

Regarding `receipt.getBase64SelfieUri()`, that is deprecated, the selfie is an attribute, it never belonged to the receipt directly.

Reply:
Okay cool, thanks for the explanation.